logger/logger.go

```diff
@@ -133,22 +133,22 @@ func Panicf(format string, values ...interface{}) {
 	logger.Panic(fmt.Sprintf(format, values...))
 }
 
-// Info logs an info message using Sprint.
+// Info logs an info message using Sprintf.
 func Info(args ...interface{}) {
```

Comment:
These methods actually aren't using `Sprintf`. That would mean we could pass tokens like `%v`, which isn't the case. There actually is an `Infof`, `Debugf`, later in the file that does indeed use `Sprintf`.

`Sprint` just takes a comma delimited list of parameters which it concatenates, while `Sprintf` leads with a format string, which is no in this function signature.